packs/astral/actions/lib/BaseAction.py

```diff
@@ -0,0 +1,28 @@
+"""
```

Comment:
Minor style thing - we use underscore separated names for modules, variable names, etc. Only use CamelCase for classes :)